File: 2019/2019W01/preppindataw01.R
library(readxl)
library(dplyr)

final <- read_excel("E:/Week 1 Input.xlsx", 
                    col_types = c("text", "numeric", "numeric", 
                                  "numeric", "numeric", "numeric", "numeric", 
                                  "skip", "skip", "skip", "skip")) %>%
  mutate('Date' = as.Date(paste(`When Sold Year`,`When Sold Month`,'01', sep='-')),
         'Total Cars Sold' = `Red Cars`+`Silver Cars`+`Black Cars`+`Blue Cars`) %>%
  select(-c('When Sold Year','When Sold Month'))

View(final)

File: 2019/2019W01/PreppindataW01.py
import pandas as pd
import re
import datetime

xls = pd.ExcelFile("E:\Week 1 Input.xlsx")
df = pd.read_excel(xls,xls.sheet_names[0])
df.drop([col for col in df.columns if re.match(r'^Unnamed',col)], axis=1, inplace=True) 
df['Date'] = df.apply(lambda x: datetime.date(x['When Sold Year'], x['When Sold Month'], 1),axis=1)
df['Total Cars Sold'] = df['Red Cars']+df['Silver Cars']+df['Black Cars']+df['Blue Cars']
df.drop(['When Sold Year','When Sold Month'], axis=1, inplace=True)